### PR TITLE
[Form, Security, Validator] Add missing Turkish translations (tr)

### LINKED ISSUE #38768: Missing translations for Turkish (tr)
Hello,

There are some missing translation for Turkish (tr). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.tr.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.tr.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.tr.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.tr.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by ahmed-bhs**
[Form, Security, Validator] Add missing Turkish translations (tr) related issue #38768
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38768
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.tr.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please provide a valid phone number.</source>
+                <target>lütfen geçerli bir telefon numarası sağlayın.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Onay kutusunda geçersiz bir değer var.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Geçerli bir e.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.tr.xlf]: Can you double check this?

**Review by Nyholm — COMMENTED** [review]
Thank you. Good work!

I just have one small question.

**Review by Nyholm — APPROVED** [review]
Thank you for your work

File: src/Symfony/Component/Form/Resources/translations/validators.tr.xlf (hunk 1 above)
Comment (ahmed-bhs) [reply src/Symfony/Component/Form/Resources/translations/validators.tr.xlf]: Done! 👍 I have just rectified it, now it's much better understandable:
please =lütfen 
Enter = girin
Valid  = geçerli
email address = e-posta adresi

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.tr.xlf]: Lovely!

**Comment by nicolas-grekas** [comment]
Thank you @ahmed-bhs.

**Pull request merged**